README.md

```diff
@@ -77,10 +76,35 @@ To install the dependencies, run:
 
 ```
 pip install -r requirements.txt
+pip install apache-beam[gcp]
 ```
 
-To execute the Planning Evaluation Framework, we have found it most convenient to
-modify PYTHONPATH and create a symbolic link to the source code, following these steps:
+Note that there is a dependency conflict between Apache Beam and Pathos. The current workaround is manually installing Apache Beam after installing the packages in `requirements.txt`, so the dependencies of Apache Beam can be satisfied.
+
+To utilize the support of distributed computing using GCP Dataflow, the following requirements have to be fulfilled beforehand:
+1. Make sure to have read/write permissions of the target Google Cloud Storage (GCS) bucket in your GCP project. Check out the [how-to guide in Google Cloud Storage](https://cloud.google.com/storage/docs/how-to) to learn more about GCS bucket.
+2. Install Cloud SDK by following the [instruction](https://cloud.google.com/sdk/docs/install). Before running the program with Dataflow, authenticate the terminal session with your google account by running the following commands in your terminal:
```

Comment:
Sorry, I'm stuck at this step.  After installing cloud SDK, I ran `gloud auth login` but always got `bash: gcloud: command not found`.  Under which dir shall I run the command?  tried several ones but they didn't work.

Reply:
Which instruction did you use? If you are using Cloudtop, you need to follow the instruction in the `Linux` tab.